cgmes/cgmes-conversion/src/test/resources/jumperTest.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF  xmlns:base="urn:uuid:" xmlns:cim="http://iec.ch/TC57/CIM100#" xmlns:md="http://iec.ch/TC57/61970-552/ModelDescription/1#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+	<md:FullModel rdf:about="urn:uuid:test">
+		<md:Model.scenarioTime>2023-01-01T00:00:00Z</md:Model.scenarioTime>
+		<md:Model.created>2023-01-01T00:00:00Z</md:Model.created>
+		<md:Model.description>test Ground objects</md:Model.description>
+		<md:Model.version>1</md:Model.version>
+		<md:Model.profile>http://entsoe.eu/CIM/EquipmentCore/3/1</md:Model.profile>
+		<md:Model.profile>http://entsoe.eu/CIM/EquipmentOperation/3/1</md:Model.profile>
+		<md:Model.modelingAuthoritySet>http://powsybl.org</md:Model.modelingAuthoritySet>
+	</md:FullModel>
+	<cim:GeographicalRegion rdf:ID="Region">
```

Comment:
Theoretically, all rdf:ID should begin with an underscore, e.g. `rdf:ID="_Region"`